content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -82,9 +82,12 @@ To set up this kind of multicluster configuration, visit our
 
 This multicluster configuration uses a single Istio control plane running on one of the clusters.
 The control plane's Pilot manages services on the local and remote clusters and configures the
-Envoy sidecars for all of the clusters. This approach works best in environments where all of
-the participating clusters have VPN connectivity so every pod in the mesh is reachable from anywhere
-else using the same IP address.
+Envoy sidecars for all of the clusters.
+
+#### Single control plane topology with VPN Connectivity
```

Comment:
Connectivity -> connectivity

Also, make it a little shorter?
```
#### Single control plane with VPN connectivity
````

Reply:
Fixed in https://github.com/istio/istio.io/pull/3290/commits/f6e4f2afa37676ea81802a4ded9b028a042a7b44